assets/ts/ui/autocomplete/plugins/algolia.ts

```diff
@@ -23,6 +23,10 @@ export default function createAlgoliaBackendPlugin(
             templates: {
               item: AlgoliaItemTemplate
             },
+            getItemUrl({ item }) {
+              const { url, _content_url } = item as AutocompleteItem;
```

Comment:
The underscore signifies that this property is private. We should change it if it's meant to be public.

Reply:
We use it in other places [as added at this location](https://github.com/mbta/dotcom/blob/master/assets/ts/ui/autocomplete/templates/algolia.tsx#L27) so I think it is fair to keep using it in this location.

Reply:
Or, we could take the time to fix it...up to you.